tests/Test.Microsoft.Identity.Core.UIAutomation/CoreMobileTestHelper.cs

```diff
@@ -102,13 +101,15 @@ private void RetryVerificationHelper(Action verification)
                 catch (ResultVerificationFailureException ex)
                 {
                     if (attempts == CoreUiTestConstants.MaximumResultCheckRetryAttempts)
-                        Assert.Fail("Could not Verify test result");
+                        throw ex;
+                        //("Could not Verify test result");
```

Comment:
Why move away from asserts and using exceptions instead? If that is the way then remove the code which should no longer be used

Reply:
This is no longer needed. Reverted